bears/general/IndentationBear.py

```diff
@@ -136,33 +151,32 @@ def get_specified_block_range(self,
                                 first encounter or left to right.
         """
         ranges = []
-        for open_indent in indent_types:
-            close_indent = indent_types[open_indent]
-            open_pos = list(self.get_valid_sequences(
+
+        open_pos = list(self.get_valid_sequences(
```

Comment:
just noted this here, might be relevant in other places: 

quoting PEP8:

```python
# Aligned with opening delimiter.
foo = long_function_name(var_one, var_two,
                         var_three, var_four)

# More indentation included to distinguish this from the rest.
def long_function_name(
        var_one, var_two, var_three,
        var_four):
    print(var_one)

# Hanging indents should add a level.
foo = long_function_name(
    var_one, var_two,
    var_three, var_four)
```

you sometimes break a line like here and then indent it way to much. We don't have an indent bear yet, look out for this :smile: